src/redux/reducers/pretender/mocks.js

```diff
@@ -0,0 +1,198 @@
+export const getApplicationsSuccess = {
+    "cohort_apps" : [
```

Comment:
Mock data! You can add anything to it, but make sure it is similar to what the API in a production environment will return